.github/test-build

```diff
@@ -1,10 +1,11 @@
 #!/bin/sh
+set -eou pipefail
 
 # Remember where we started; this is where our source checkout lives.
-src=`pwd`
+src=$(pwd)
 
 # Load the module command and use it to load MPI.
-. /etc/profile.d/modules.sh
+./etc/profile.d/modules.sh
```

Comment:
This is not correct? `. script` is equivalent to `source script`. Is there even a `./etc` directory that exists?

Reply:
You're right - let me fix this again.